docs/source/agent-framework/historian-agents/historian-framework.rst

```diff
@@ -67,6 +67,9 @@ All Historians support the following settings:
         # Defaults to no limit.
         "backup_storage_limit_gb": 8.0,
 
+        # How full should the backup storage be for an alert to be raised
```

Comment:
is 0.9 a percentage of fullness?  What does the 0.9 mean?

Reply:
It is the decimal representation of percentage full. Updated documentation